PIG_ZOMBIE bad for 1.16.1

### Description
#### Describe the bug
<!-- A clear and concise description of the problem you're encountering. -->
On visiting island
```
[11:17:46] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to BentoBox v1.14.0-SNAPSHOT-b1816
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:319) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:600) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:537) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.addEntity0(WorldServer.java:911) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.addEntity(WorldServer.java:841) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:193) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:133) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:117) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.lambda$9(ChunkProviderServer.java:394) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at java.util.ArrayList.forEach(ArrayList.java:1540) ~[?:?]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.tickChunks(ChunkProviderServer.java:378) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.tick(ChunkProviderServer.java:346) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:329) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1107) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:354) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1010) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:849) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:165) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoSuchFieldError: PIG_ZOMBIE
        at world.bentobox.extramobs.listeners.MobsSpawnListener.onEntitySpawn(MobsSpawnListener.java:71) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor127.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:315) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        ... 21 more
[11:17:48] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to BentoBox v1.14.0-SNAPSHOT-b1816
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:319) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:600) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:537) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.addEntity0(WorldServer.java:911) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.addEntity(WorldServer.java:841) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:193) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:133) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.SpawnerCreature.a(SpawnerCreature.java:117) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.lambda$9(ChunkProviderServer.java:394) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at java.util.ArrayList.forEach(ArrayList.java:1540) ~[?:?]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.tickChunks(ChunkProviderServer.java:378) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.ChunkProviderServer.tick(ChunkProviderServer.java:346) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:329) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1107) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:354) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1010) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:849) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:165) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoSuchFieldError: PIG_ZOMBIE
        at world.bentobox.extramobs.listeners.MobsSpawnListener.onEntitySpawn(MobsSpawnListener.java:71) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor127.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:315) ~[spigot-1.16.1.jar-2020-06-25-0558:git-Spigot-758abbe-3f0c333]
        ... 21 more
```

#### Steps to reproduce the behavior
<!-- Step-by-step instructions for us to reproduce the bug on our side. -->
Install Spigot 1.16.1 (replacing 1.15.2)
Visit island

#### Expected behavior
<!-- What do you think should happen? -->

### Environment

##### Output of `/bbox version` (Mandatory)
<!-- Paste the `/bbox version` output inside the code block below (remove the underscores). Do not provide an image. -->
```
>bbox version
[11:20:34] [Server thread/INFO]: Running SPIGOT Invalid.
[11:20:34] [Server thread/INFO]: BentoBox version: 1.14.0-SNAPSHOT-b1816
[11:20:34] [Server thread/INFO]: Database: JSON
[11:20:34] [Server thread/INFO]: Loaded Game Worlds:
[11:20:34] [Server thread/INFO]: world_skyblock (SkyBlock): Overworld, [nether_color]Nether, [end_color]End
[11:20:34] [Server thread/INFO]: Loaded Addons:
[11:20:34] [Server thread/INFO]: Biomes 1.13.0-SNAPSHOT-b161 (ENABLED)
[11:20:34] [Server thread/INFO]: BSkyBlock 1.13.0-SNAPSHOT-b601 (ENABLED)
[11:20:34] [Server thread/INFO]: Challenges 0.9.0-SNAPSHOT-b418 (ENABLED)
[11:20:34] [Server thread/INFO]: ExtraMobs 1.11.0.2-SNAPSHOT-b4 (ENABLED)
[11:20:34] [Server thread/INFO]: Level 2.1.0-SNAPSHOT-b365 (ENABLED)
```

--- a/pom.xml
+++ b/pom.xml
@@ -35,7 +35,7 @@
         <java.version>1.8</java.version>
         <powermock.version>1.7.4</powermock.version>
         <!-- More visible way how to change dependency versions -->
-        <spigot.version>1.15.2-R0.1-SNAPSHOT</spigot.version>
+        <spigot.version>1.16.1-R0.1-SNAPSHOT</spigot.version>
         <bentobox.version>1.11.0</bentobox.version>
         <!-- Revision variable removes warning about dynamic version -->
         <revision>${build.version}-SNAPSHOT</revision>
@@ -45,7 +45,7 @@
                 X.Y.Z -> BentoBox core version
                 .M    -> Addon development iteration.
         -->
-        <build.version>1.11.0.2</build.version>
+        <build.version>1.11.0.3</build.version>
         <build.number>-LOCAL</build.number>
     </properties>
 

--- a/src/main/java/world/bentobox/extramobs/listeners/MobsSpawnListener.java
+++ b/src/main/java/world/bentobox/extramobs/listeners/MobsSpawnListener.java
@@ -68,7 +68,8 @@ public class MobsSpawnListener implements Listener
 		}
 
 
-		if (event.getEntityType() == EntityType.PIG_ZOMBIE &&
+		if ((event.getEntityType().name().equals("PIG_ZOMBIE") ||
+			event.getEntityType().name().equals("ZOMBIFIED_PIGLIN")) &&
 			this.addon.getPlugin().getIWM().isIslandNether(world))
 		{
 			// replace pigmen with blaze or wither

